tst/testinstall/read.tst

```diff
@@ -77,6 +79,12 @@ gap> FileString( Filename(dir, "test.g.gz"), "\037\213\b\b0,\362W\000\ctest.g\00
 32
 gap> StringFile( Filename(dir, "test.g") ) = "1+1;\n" or ARCH_IS_WINDOWS(); # works only when Cygwin installed with gzip
 true
+gap> StringFile( "/" );
+Error, in StringFile: Is a directory (21)
+
+gap> FileString("/dev/full", "outdata", false);
+Error, in FileString: No space left on device (28)
```

Comment:
Unfortunately, this test fails on OS X (and I am kinda surprised it works on cygwin).